index.html

```diff
@@ -70,10 +70,13 @@ <h1>GAMERS</h1>
                 <a class="nav-link" href="cart.html">Cart </a>
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="#">Shop</a>
+                <a class="nav-link" href="shipping.html">Shipping</a>
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="#">Contact</a>
+                <a class="nav-link" href="contact.html">Contact</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="client.html">About</a>
               </li>
```

Comment:
font is not like the design

Reply:
spaces between links not like the design